drivers/spi/spi_ll_stm32.c

```diff
@@ -121,8 +119,12 @@ static void spi_stm32_shift_s(SPI_TypeDef *spi, struct spi_stm32_data *data)
 			spi_context_update_tx(&data->ctx, 2, 1);
 		}
 	}
+	else
```

Comment:
Always put {} on if/for/while/else etc... even on 1 liners statement then.

Reply:
Damn... ok